src/middlewares/transform.middleware.ts

```diff
@@ -0,0 +1,58 @@
+import { NextFunction, Request, Response } from 'express';
+import { dayjs } from '@utils/dayjs.util';
+
+export const transform = () => {
+  return (body: Object, req: Request, res: Response) => {
+    const timestamp = dayjs().utc().format();
+
+    if (Array.isArray(body)) {
+      let page = Number(req.query.page);
```

Comment:
Pagination padding should be done in the middleware before it hits the controller, and not after